opentelemetry-example/src/main/scala/zio/telemetry/opentelemetry/example/ProxyApp.scala

```diff
@@ -1,23 +1,20 @@
 package zio.telemetry.opentelemetry.example
 
-import sttp.client3.asynchttpclient.zio.AsyncHttpClientZioBackend
+import zio._
 import zio.config.magnolia._
 import zio.config.typesafe.TypesafeConfig
-import zio.telemetry.opentelemetry.example.config.AppConfig
-import zio.telemetry.opentelemetry.example.http.{ Client, ProxyHttpApp, ProxyHttpServer }
-import zio._
+import zio.http.ZClient
 import zio.telemetry.opentelemetry.baggage.Baggage
 import zio.telemetry.opentelemetry.context.ContextStorage
+import zio.telemetry.opentelemetry.example.config.AppConfig
+import zio.telemetry.opentelemetry.example.http.{ Client, ProxyHttpApp, ProxyHttpServer }
 import zio.telemetry.opentelemetry.tracing.Tracing
 
 object ProxyApp extends ZIOAppDefault {
 
   private val configLayer = TypesafeConfig.fromResourcePath(descriptor[AppConfig])
 
-  private val httpBackendLayer: TaskLayer[Backend] =
-    ZLayer.scoped {
-      ZIO.acquireRelease(AsyncHttpClientZioBackend())(_.close().ignore)
-    }
+  private val httpBackendLayer: TaskLayer[Backend] = ZClient.default
```

Comment:
I don't think we need this extra variable here anymore, it would be better to pass `ZClient.default` directly into `.provide`

Reply:
fixed